src/components/BackOffice/CloseWorkOrders/utils.js

```diff
@@ -0,0 +1,32 @@
+export const formatWorkOrderReferences = (workOrderReferences) => {
+  return workOrderReferences
+    .split('\n')
+    .map((x) => x.trim())
+    .filter((x) => x)
```

Comment:
is this intended to remove duplicates. if so, have you tested that this is the case?

Reply:
It doesn't remove duplicates, but that's definitely worth adding . This takes the contents from the textArea, and returns an array of workOrderReferences. The filter removes nulls